contrasts_kwdyz11.qmd

```diff
@@ -78,19 +86,31 @@ Controlling the ordering of levels for contrasts:
 ## SeqDiffCoding
 
 The `SeqDiffCoding` contrast corresponds to `MASS::contr.sdif()` in R.
+The assignment of random factors such as `Subj` to `Grouping()` is necessary when the sample size is very large. We recommend to include it always, but in this tutorial we do so only in the first example.
```

Comment:
not necessary anymore -- we do it automatically!

Reply:
Replace "is necessary when the sample size is very large"  with. "replace is optional."?

Reply:
"It could be included to explicitly document grouping variables."